index.js

```diff
@@ -13,6 +13,9 @@ export default function toCSS(obj) {
       value = `${value}px`;
     }
 
-    return `${key}:${value} !important;`;
+    const isTouch = ('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch;
```

Comment:
I think that still suffers from the issue I described before. 

The article linked in the code goes through the challenges surrounding this particular thing.

This section in particular covers the issue I was referring to: http://www.stucox.com/blog/you-cant-detect-a-touchscreen/#an-api-for-an-api